### ISSUE TITLE
Feature request - Inner Favorites bar (Vertical tool bar)

### ISSUE THREAD

**Issue by aca027** [tablacus/TablacusExplorer#690]
First of all, praise for the program. All I can say is that it is phenomenal.

As seen in the title, the question is whether it is possible to make a "vertical favorite bar" for both/all panels (Inner Favorites bar)?

Thank you!!!

**Comment by duarteframos** [comment]
https://github.com/tablacus/TablacusExplorerAddons/wiki/favoritesbar

**Comment by aca027** [comment]
It is for one panel. If possible, that each panel has its own vertical bar.

**Comment by duarteframos** [comment]
I'm not aware of any method, but if there was any, they both show the same thing, since there is only one central favorites storage

**Comment by aca027** [comment]
True, but every time the shortcut is started, the panel must be activated first. for a larger number of operations, quite a lot of clicks or shortcuts on the keyboard.

**Comment by ccfs** [comment]
A workaround is putting "Favorites" add-on in an "Internal" bar.

**Comment by aca027** [comment]
I know about that option, but it requires extra clicks again.
Is it possible to create an "Inner Favorite bar" (vertical bar) for each panel?

**Comment by ccfs** [comment]
I am not the application owner, I can't tell.
Even if it is possible, hugh effort will be required, since currently there is no "Internal left bar" nor "Internal right bar".

**Comment by ccfs** [comment]
You may try these modified versions of add-ons:
- [Favorites](https://github.com/tablacus/TablacusExplorer/files/13060333/favorites_106.mouse.over.zip): The menu shows up when the mouse pointer is on the button. No need to click.
- [Tab plus](https://github.com/tablacus/TablacusExplorer/files/13060334/tabplus_196.fav.menu.zip): Favorites menu shows up when the mouse pointer is on the + button. "Favorites" add-on, either the original or the modified version, is required.

**Comment by aca027** [comment]
Thanks for the answers. 
Should this post remain open so that someone in charge of changes to the program can do something about this issue or should it be closed?

**Comment by ccfs** [comment]
I suggest to implement the enhancement of Tab plus add-on alone, which is good enough for use.
You can keep this issue opened until the enhancement is made.

**Comment by aca027** [comment]
![big-thumbs-up_50](https://github.com/tablacus/TablacusExplorer/assets/148001229/efd7c05b-bf60-4dd8-aae0-28583ca17253)

**Comment by aca027** [comment]
At your suggestion I started using the "favorites mouse over" add-on, and it is quite good.
Is it possible to create a favorites menu without entering text in the part related to the name?
In the picture below, instead of the name, I entered an empty space, because if nothing is entered, then there is no icon in the menu:
![favorites_final](https://github.com/tablacus/TablacusExplorer/assets/148001229/75286f38-c67f-4221-b13a-c5f39ad315f7)
Without the fields marked in red, intended for the name, the menu would look more compact.

**Comment by ccfs** [comment]
Sorry I don't have a solution to this.

**Comment by aca027** [comment]
It seems to me that I am asking too much. 
Thanks for the support so far.

**Comment by ccfs** [comment]
The [patched version](https://github.com/tablacus/TablacusExplorer/issues/692) of Horizontal favorites bar now can:
- be placed in an inner position.
- hide button names by an option.

**Comment by aca027** [comment]
This is good. We are getting closer...
